curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65364566e84e378837fbaf2a.md

```diff
@@ -7,9 +7,10 @@ dashedName: step-33
 
 # --description--
 
-You need to hook up the `playSong` function to a `click` event listener so the songs will start playing when the play button is clicked.
 
-Add a click event listener to the `playButton` element, then use arrow syntax to pass in a callback with an empty pair of curly braces.
+In previous steps you built out the functionality for playing a song. Now you need to add the functionality to the play button so that it will play the current song when it is clicked on.
+
+Use the `addEventListener()` method and pass in the string `click` for the first argument as the event type to listen for and an empty callback function with arrow syntax for the second argument. Ex.(`()=>{}`).
```

Comment:
```suggestion
Use the `addEventListener()` method and pass in the string `click` for the first argument as the event type to listen for and an empty callback function with arrow syntax for the second argument, e.g., `() => {}`.
```

Bit of a nitpick, but I would prefer e.g. instead of ex. here and removing the parentheses around the example.